src/simulation/openables/BronzeHAMChest.ts

```diff
@@ -0,0 +1,27 @@
+import LootTable from '../../structures/LootTable';
+import SimpleOpenable from '../../structures/SimpleOpenable';
+import { rand } from '../../util';
+
+const rolls = rand(1, 5);
+
+/* Gem loots */
+
+//TODO
```

Comment:
write TODOs that actually say something please

ie. `// TODO: check back for wiki drop table update` from another one in the repo